libtransmission/torrent-metainfo.cc

```diff
@@ -31,108 +31,6 @@
 
 using namespace std::literals;
 
-//// C BINDINGS
```

Comment:
removing this seems off-topic for this PR

Reply:
OK, I can revert that part. Although it was dead code and I wasted time looking at its `tr_saveFile` before realising that it wasn't even compiling.